tilequeue/command.py

```diff
@@ -1070,6 +1070,84 @@ def tilequeue_dump_tiles_of_interest(cfg, peripherals):
     )
 
 
+def tilequeue_load_tiles_of_interest(cfg, peripherals):
+    """
+    Given a newline-delimited file containing tile coordinates in
+    `zoom/column/row` format, load those tiles into a new Redis SET
+    alongside the existing one, then RENAME the new to the old.
+    """
+    logger = make_logger(cfg, 'load_tiles_of_interest')
+    logger.info('Loading tiles of interest')
+
+    toi_filename = "/Users/iandees/Downloads/tile_requests_unique_noeurope.txt"
```

Comment:
I'm not sure how to fix this. See [this thread above](https://github.com/tilezen/tilequeue/pull/122#discussion_r88307679), but I don't really want to add a thing to the config file. I'd like to accept the filename at the command line.